debug_toolbar/apps.py

```diff
@@ -206,3 +207,23 @@ def js_mimetype_check(app_configs, **kwargs):
             )
         ]
     return []
+
+
+@register()
+def debug_toolbar_installed_when_running_tests_check(app_configs, **kwargs):
+    """
+    Check that the Django Debug Toolbar is installed and works well with the current tests
+    """
+    if (
+        settings.DEBUG is False
+        and dt_settings.get_config()["RUNNING_TESTS"] == "test" in sys.argv
```

Comment:
you don't need == "test" in sys.argv because you set already in debug_toolbar/settings.py:46 (if you print it you can check the value) also you can just put "not settings.DEBUG" it's already a bool too.

Reply:
Let me test this out?